doc/module_examples/03_action_required/check.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+. /usr/share/preupgrade/common.sh
+
+#END GENERATED SECTION
+
+#
+# The whole section above is processed and modified by the preupg-xccdf-compose
+# script, according to the contents of the INI file (in this case it is the
+# content.ini file). In addition, the LICENSE used by the Preupgrade Assistant
+# is inserted.
+#
+
+##
+# Briefly:
+#   In case a manual inspection, check or action BEFORE the upgrade is needed,
+#   do the following things:
+#     1) provide a text in $SOLUTION_FILE: a description of the problem and
+#        remediation instructions
+#     2) use "log_high_risk" to provide a short message that a problem was
```

Comment:
Yes - it has to be log_high_risk - that's how PA resolves it. That's why I think it is good to write there - because it is not intuitive.